www/js/search.js

```diff
@@ -931,7 +944,9 @@ module.exports = {
         'data-line-count': line.lineCount,
       },
       h(
-        `a#line${line.ID}.panktee.${englishHeading ? 'roman' : 'gurmukhi'}${seenClasses}`,
+        `a#line${line.ID}.cpid-${line.crossPlatformID}.panktee.${
```

Comment:
why is cpid-${line.crossPlatformID} being added as a class and not as a `data-attribute`?